File: datalake/src/main/java/com/sequenceiq/datalake/service/registry/ServiceAddressResolvingException.java
package com.sequenceiq.datalake.service.registry;

public class ServiceAddressResolvingException extends Exception {
    public ServiceAddressResolvingException(String message) {
        super(message);
    }

    public ServiceAddressResolvingException(String message, Throwable cause) {
        super(message, cause);
    }

    public ServiceAddressResolvingException(Throwable cause) {
        super(cause);
    }
}

File: core/src/main/java/com/sequenceiq/cloudbreak/service/registry/ServiceAddressResolvingException.kt
package com.sequenceiq.cloudbreak.service.registry

class ServiceAddressResolvingException : Exception {
    constructor(message: String) : super(message) {
    }

    constructor(message: String, cause: Throwable) : super(message, cause) {
    }

    constructor(cause: Throwable) : super(cause) {
    }
}
